src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,26 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
+import org.hibernate.SessionFactory;
 
 public class MovieServiceImpl implements MovieService {
+    private final MovieDao dao;
+
+    public MovieServiceImpl(SessionFactory sessionFactory) {
+        dao = new MovieDaoImpl(sessionFactory);
+    }
```

Comment:
```suggestion
    public MovieServiceImpl(MovieDao movieDao) {
        this.movieDao = movieDao;
    }
```

Reply:
Let's better to have dependency on `MovieDao` interface, than `MovieDaoImpl` implementation